planning/scenario_planning/common/motion_velocity_smoother/README.md

```diff
@@ -86,13 +86,13 @@ After the optimization, a resampling called `post resampling` is performed befor
 
 ### Input
 
-| Name                                       | Type                                | Description                   |
-| ------------------------------------------ | ----------------------------------- | ----------------------------- |
-| `~/input/trajectory`                       | `autoware_planning_msgs/Trajectory` | Reference trajectory          |
-| `/planning/scenario_planning/max_velocity` | `std_msgs/Float32`                  | External velocity limit [m/s] |
-| `/localization/odometry`                   | `nav_msgs/Odometry`                 | Current odometry              |
-| `/tf`                                      | `tf2_msgs/TFMessage`                | TF                            |
-| `/tf_static`                               | `tf2_msgs/TFMessage`                | TF static                     |
+| Name                                       | Type                                     | Description                   |
+| ------------------------------------------ | ---------------------------------------- | ----------------------------- |
+| `~/input/trajectory`                       | `autoware_auto_planning_msgs/Trajectory` | Reference trajectory          |
+| `/planning/scenario_planning/max_velocity` | `std_msgs/Float32`                       | External velocity limit [m/s] |
+| `/localization/odometry`                   | `nav_msgs/Odometry`                      | Current odometry              |
```

Comment:
want to modify here too `localization/kinematic_state` code modify PR can be later

Reply:
Oh I couldn't find that, thanks!

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/576/commits/cfb7d2d87ec9f4479f6ee74422af82100868a3dd
DONE

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/576/commits/3167bf631f3a79661cacfb569d5ea0cdf98feda3